src/PermissionRegistrar.php

```diff
@@ -217,12 +217,9 @@ public function getPermissions(array $params = [], bool $onlyOne = false): Colle
         return $permissions;
     }
 
-    /**
-     * Get an instance of the permission class.
-     */
-    public function getPermissionClass(): Permission
+    public function getPermissionClass(): string
     {
-        return app($this->permissionClass);
+        return $this->permissionClass;
```

Comment:
I can't verify that, it shouldn't, I think that originally it should have been
```php
return app(PermissionContract::class);
```
Because there is a binding
https://github.com/spatie/laravel-permission/blob/f7582f809555706e860512eb809509776dfaff33/src/PermissionRegistrar.php#L232
Where `Permission::class` is the contract
https://github.com/spatie/laravel-permission/blob/f7582f809555706e860512eb809509776dfaff33/src/PermissionRegistrar.php#L11

Reply:
but as I said in https://github.com/spatie/laravel-permission/discussions/2367#discussioncomment-5382764
>we have a string `$permissionClass` propertie, but on `getPermissionClass()` package set an Permission contract instance on that propertie

`app($this->permissionClass)` is returning an instance, but method name tells me that it returns the property 'private $permissionClass' that is as a string

Reply:
@drbyte i did search swoole commits, i only find 

- https://github.com/spatie/laravel-permission/commit/5f0429b64e22b2e1fea3fbfb45ccfc2a92504a5d
- https://github.com/spatie/laravel-permission/commit/cff803fb54ff6aaf395e1c32a9bdaa77ecac3286

Reply:
👍